css/style.css

```diff
@@ -1,6 +1,21 @@
+.navbar-inner {
```

Comment:
General comment: a commit message like "Rework the CKAN Explorer" makes it tough to know what exactly you were doing. Is it a refactor, is it adding features etc etc.

Reviewing the changes below does not make this super clear ;-0 -- my guess is this is more refactor ...